src/Channel/ServiceChannel.cpp

```diff
@@ -38,7 +38,12 @@ ServiceChannel::ServiceChannel(boost::asio::io_service::strand& strand,
 
 void ServiceChannel::send(messenger::Message::Pointer message, SendPromise::Pointer promise)
 {
+#if BOOST_VERSION < 106600
```

Comment:
so `context` doesnt exist in older boost? would like to minimize any conditionals like this

Reply:
context got just got renamed.  Yaay for API stability I guess.  The later boost used by buildroot won't work without the conditional.  boost 1.66 is the first appearance of the new name and the deprecation of the existing name.

Reply:
ugh not a fan but i guess what can you do